hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -618,20 +620,35 @@ cdef class PureMarketMakingStrategy(StrategyBase):
             ExchangeBase market = self._market_info.market
             list buys = []
             list sells = []
-        for level in range(0, self._buy_levels):
-            price = self.get_price() * (Decimal("1") - self._bid_spread - (level * self._order_level_spread))
-            price = market.c_quantize_order_price(self.trading_pair, price)
-            size = self._order_amount + (self._order_level_amount * level)
-            size = market.c_quantize_order_amount(self.trading_pair, size)
-            if size > 0:
-                buys.append(PriceSize(price, size))
-        for level in range(0, self._sell_levels):
-            price = self.get_price() * (Decimal("1") + self._ask_spread + (level * self._order_level_spread))
-            price = market.c_quantize_order_price(self.trading_pair, price)
-            size = self._order_amount + (self._order_level_amount * level)
-            size = market.c_quantize_order_amount(self.trading_pair, size)
-            if size > 0:
-                sells.append(PriceSize(price, size))
+        order_override = self._order_override
+        if order_override is not None and len(order_override) > 0:
+            for key, value in order_override.items():
+                if str(value[0]) in ["buy", "sell"]:
+                    if str(value[0]) == "buy":
+                        price = self.c_get_mid_price() * (Decimal("1") - Decimal(value[1]))
```

Comment:
Do you need to convert the percentage value to the actual decimal value here? A user enter 1, but this means 0.01 spread.

Reply:
Yes, I changed the code to first convert percentage value to the actual decimal value, i.e. / Decimal("100")